db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Answer.delete_all
+ApplicationRecord.delete_all
+Keyword.delete_all
+Officer.delete_all
+Question.delete_all
+Rso.delete_all
+
+rso.create!(name: 'Association for Computer Machinery',
+            shortName: 'ACM',
+            category: 'Computer Science')
+
+rso.create!(name: 'Knitting Club',
```

Comment:
Those keywords are something we're going to have to figure out at somepoint but not now. They can be very general things like you mentioned. 

Outdoors, Education, Activism, LGBT, etc. idk